pallets/dactr/src/lib.rs

```diff
@@ -441,6 +554,7 @@ pub mod pallet {
 			max_block_size: Option<u64>,
 			max_total_old_submission_size: Option<u64>,
 			disable_old_da_submission: Option<bool>,
+			vouch_threshold: Option<u32>,
```

Comment:
This has too many params already. It would make more sense to group them all together and have them as fields of a struct.